### ISSUE TITLE
bid, ask and price incorrect in api

### ISSUE THREAD

**Issue by forReason** [dougdellolio/coinbasepro-csharp#217]
I am having issues of the Prices beeing wrong coming from GetProductTicketAsync

This is my code:
`var response = await ProClient.ProductsService.GetProductTickerAsync(CoinbasePro.Shared.Types.ProductType.BtcEur);`

the askprice which comes back is 100000

**Comment by dougdellolio** [comment]
hi @forReason . are you using sandbox or live servers? I am seeing correct pricing on live but sandbox usually returns 100000.

**Comment by forReason** [comment]
ahh thank you. I used Sandbox.
Also, what is the "unsignedSignature" in the Authenticator?
I have set it like this, I hope its correct:
`private static protected Authenticator authenticator = new Authenticator(Credentials.ProApiKey, Credentials.ProApiSecret, Credentials.ProPassphrase);`
I was used the secret for unsigned signature and for the pasphrase i had to create a second api key because it is only shown at generation time.
